common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -112,6 +112,7 @@ class VideoModule(VideoFields, VideoTranscriptsMixin, VideoStudentViewHandlers,
     #TODO: For each of the following, ensure that any generated html is properly escaped.
     js = {
         'js': [
+            resource_string(module, 'js/src/time.js'),
```

Comment:
Sorry @e-kolpakov , I should have been more explicit.

`js/src/time.js` file used to be included by [DiscussionModule](https://github.com/edx/edx-platform/pull/12582/files#diff-6e99382019ec7061fa7a155bea7d251fL59), even though it wasn't required there (maybe it used to be?).  So when the Discussion XBlock didn't include it, the Video XModule tests started failing.